### PR TITLE
Network Firewall docs (issue #1025)

### PR DESCRIPTION
**Pull request by harlo**
as per https://github.com/freedomofpress/securedrop/issues/1025

### CODE REVIEW

**Comment by garrettr** [comment]
:+1: What do you think @dolanjs @conorsch?

File: docs/network_firewall.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Leave the defaults for "Time Server Information". Click Next.
 
 On "Configure WAN Interface", enter the appropriate configuration for your network. Consult your local sysadmin if you are unsure what to enter here. For many environments, the default of DHCP will work and the rest of the fields can be left blank. Click Next.
 
-For "Configure LAN Interface", set the IP address and subnet mask of the Application Subnet for the LAN interface. Click Next.
+For "Configure LAN Interface", set the IP address and subnet mask of the Application Subnet for the LAN interface.  Be sure that the CIDR prefix correctly corresponds to your subnet mask-- pfsense should automatically calculate this for you, but you should always check. Click Next.
```
Comment (dolanjs) [inline docs/network_firewall.md]: Should we add that for the values used in the doc the CIDR should be `/24` So they don't need to look it up?

**Comment by harlo** [comment]
...maybe we add something like, "default will always be `/24`"?  are other configs supported?

File: docs/network_firewall.md (hunk 1 above)
Comment (conorsch) [inline docs/network_firewall.md]: :+1: Furthermore, in L36, I suggest: `This is a very common subnet choice for home routers.` -> `The /24 subnet is a common choice for home routers.` Specifically referring to the subnet by CIDR notation adds clarity for non-technical folks, at no cost to clarity for technical audiences.

**Comment by dolanjs** [comment]
In the install docs we recommend using  the 10.20.2.0 network with a subnet mask 255.255.255.0 for the monitor server's network. `/24` is the CIDR notation for the 255.255.255.0 subnet mask.

It is configurable by the admin to accommodate certain edge cases. One of which is if the 10.20.1.0 or 10.20.2.0 networks are in use someplace else in the corporate environment. Then the admin would want to use a different network(s) for the app and monitor server networks to avoid routing table issues with SD network firewall.

Any internal ip range netmask that provides 2 usable IP addresses will work.

**Comment by conorsch** [comment]
One more change: In L36 `This is a very common subnet choice for home routers.` -> `The /24 subnet is a common choice for home routers.` Specifically referring to the subnet by CIDR notation adds clarity for non-technical folks, at no cost to clarity for technical audiences.

Then we should be all set.

**Comment by garrettr** [comment]
Agree with @conorsch's last comment. One of the best pieces of advice about writing clearly I've ever received is to always carefully consider the use of the word "this". It may seem clear to the writer at the time of writing due to their mental context, but it is often ambiguous and confusing!

**Comment by garrettr** [comment]
lgtm. Ready to merge @harlo?

**Pull request merged**